.github/actions/integration-tests/mv-params/oslat-2.json

```diff
@@ -0,0 +1,16 @@
+{
+    "global-options": [
+	{
+	    "name": "required",
+	    "params": [
+                { "arg": "duration", "vals": [ "10", "15" ], "role": "client" },
+	        { "arg": "rtprio", "vals": [ "2" ], "role": "client" }
```

Comment:
is it a testcase rtprio=2 ?

Reply:
It is debatable whether or not we would want to actually run this "for real", I was just trying to exercise the different options with varying parameters for some differences between the iterations.

Note, I wouldn't expect any of the results from these tests to be good.